src/Microsoft.ML.Data/Transforms/NormalizeColumnDbl.cs

```diff
@@ -932,7 +932,7 @@ public ImplVec(IHost host, TFloat[] mean, TFloat[] stddev, bool useLog)
 
                     public static ImplVec Create(ModelLoadContext ctx, IHost host, VectorType typeSrc)
                     {
-                        host.Check(typeSrc.ItemType.RawType == typeof(TFloat), "The column type must be vector of R8.");
+                        host.Check(typeSrc.ItemType.RawType == typeof(TFloat), "The column type must be vector of Double.");
```

Comment:
>vector [](start = 104, length = 6)

Here's an example of a place where we've left it lowercase `vector`. Not sure what approach you want to take here.

Reply:
lowercase

---
In reply to: [268028513](https://github.com/dotnet/machinelearning/pull/3046#discussion_r268028513) [](ancestors = 268028513)

Reply:
Good, I agree, thank you.

---
In reply to: [268267564](https://github.com/dotnet/machinelearning/pull/3046#discussion_r268267564) [](ancestors = 268267564,268028513)